src/Sulu/Bundle/ContactBundle/Resources/config/lists/accounts.xml

```diff
@@ -247,12 +254,25 @@
             <field-name>id</field-name>
             <entity-name>SuluTagBundle:Tag</entity-name>
 
-            <joins>
-                <join>
-                    <entity-name>SuluTagBundle:Tag</entity-name>
-                    <field-name>%sulu.model.account.class%.tags</field-name>
-                </join>
-            </joins>
+            <joins ref="tags"/>
         </group-concat-property>
+
+        <property
+            name="tagId"
+            visibility="never"
```

Comment:
it is important that this property will never be included in the `SELECT` statement, otherwise accounts with multiple tags will be duplicated in the response